### ISSUE TITLE
PPO starts learning and loses everything learned

### ISSUE THREAD

**Issue by m-kul** [Unity-Technologies/ml-agents#1237]
Hello,

When I try to train the rewards go higher for a while then drops suddenly. What may be the cause of this? I am using PPO.

This is my training config:
    max_steps: 5.0e8
    learning_rate: 1e-3
    batch_size: 320
    num_epoch: 3
    buffer_size: 2000
    beta: 1.0e-2
    hidden_units: 320
    summary_freq: 2000
    time_horizon: 128
    num_layers: 3
    normalize: false

<img width="718" alt="screen shot 2018-09-16 at 12 14 53" src="https://user-images.githubusercontent.com/15053537/45596345-3041d600-b9aa-11e8-84fb-f23bd1ad6c65.png">

**Comment by MarcoMeter** [comment]
Hi m-kul
you've got some crazy high hyperparameters setup and espeically a really high learning rate.
Without knowledge on your observation space, action space, the complexity of your environment and its rewards, it is not easy to give you helpful advice.

**Comment by m-kul** [comment]
Hi @MarcoMeter ,

I am trying to train a similar setup like soccer in the examples. I have 60 vectors, 56 of them are raycasts and 4 of them are for having the ball passing and stuff. I have 2 agents working together to throw the ball to a hole.

**Comment by eshvk** [comment]
@m-kul Do you still need additional help, if so, can you please post code?

**Comment by xiaomaogy** [comment]
Thanks for reaching out to us. Hopefully you were able to resolve your issue.  We are closing this due to inactivity, but if you need additional assistance, feel free to reopen the issue.
